vuepress/docs/v6.3.2/tutorials/customize-the-platform/pda-technical-guide.md

```diff
@@ -0,0 +1,221 @@
+      "type": "object",
+      "properties": {
+        "Application": {
+          "title": "Application",
+        "type": "object",
+        "required": [],
+        "properties": {
+            "mortgageamount": {
+              "type": "integer",
+              "title": "Mortgage amount",
+              "description": "Mortgage amount"
+            },
+            "downpayment": {
+              "type": "integer",
+              "title": "Down Payment",
+              "description": "Down Payment"
+            },
+            "applicant": {
+              "title": "Applicant",
+              "type": "object",
+              "required": [],
+              "properties": {
+                "name": {
+                  "type": "string",
+                  "title": "Name",
+                  "description": "Name",
+                  "maxLength": 100
+                },
+                "annualincome": {
+                  "type": "integer",
+                  "title": "Annual Income",
+                  "description": "Annual Income"
+                }
+              }
+            },
+            "property": {
+              "title": "Property",
+              "type": "object",
+              "required": [],
+              "properties": {
+                "age": {
+                  "type": "integer",
+                  "title": "Age of property",
+                  "description": "Age of property"
+                },
+                "address": {
+                  "type": "string",
+                  "title": "Address of property",
+                  "description": "Address of property",
+                  "maxLength": 100
+                    }
+              }
+            }
+          }
+        },
+        "inlimit": {
+          "type": "boolean",
+        "title": "Is mortgage application in limit?"
+        }
+      }
+    }
+ 
+From this JSON (you can copy & paste it into the `react-jsonschema-form`
+sandbox) we can see that there is a main form with a title “Mortgage
+Application Form." The root form `Mortgage Application Form` has two
+properties: one is a subform called `Application` and the other is a
+checkbox field (field ID is `inlimit`).
+ 
+The `Application` subform contains two fields (`Mortgage Amount` with field ID `mortgageamount` and `Down Payment` with field ID `downpayment`) and two subforms (`Applicant` with field ID `applicant` and `Property` with field ID `property`).
+ 
+The `Applicant` subform contains two fields (`Name` with field ID `name` and `Annual Income` with field ID `annualincome`). The `Property` subform also contains two fields (`Age of property` with field ID `age` and `Address of property` with field ID `address`).
+ 
+By default (without providing UI schema), these are listed as one field per row. To use Entando’s implementation of Grid layout, users have to
+provide UI schema with details about each field. For example, if we
+would like to have a layout that looks like this (fields are marked
```

Comment:
haha most of the language on this page is inherited. in this case i kept it wordy because i'm not sure if the detail is necessary/valuable, like if Grid layout is different from any old layout. if so we should keep it as is... @nshaw??

Reply:
Per our slack, unless there's something egregious let's leave the pda-technical-guide alone. It's all inherited and not worth reworking too heavily for now.